<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Requirement</class>
 <widget class="QWidget" name="Requirement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Requirement</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>List of requirements</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QListWidget" name="p_list_Requirements">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="p_button_AddRequirement">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="p_button_DeleteRequirement">
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="minimumSize">
          <size>
           <width>360</width>
           <height>340</height>
          </size>
         </property>
         <property name="title">
          <string>Single Requirement</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="7" column="3">
           <widget class="QCheckBox" name="p_checkbox_Neglect">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="6" column="3">
           <widget class="QCheckBox" name="p_checkbox_BiggerAmountWorkers">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="p_lbl_EndDate">
            <property name="text">
             <string>End date</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2" colspan="2">
           <widget class="QSpinBox" name="p_spinbox_Severity">
            <property name="maximum">
             <number>63</number>
            </property>
           </widget>
          </item>
          <item row="6" column="0" colspan="2">
           <widget class="QLabel" name="p_lbl_BiggerAmountWorkers">
            <property name="text">
             <string>Allow bigger amount workers</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QLabel" name="p_lbl_NumberWorkers">
            <property name="text">
             <string>Number of workers</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="p_lbl_StartDate">
            <property name="text">
             <string>Start date</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="3">
           <widget class="QLineEdit" name="p_line_ID"/>
          </item>
          <item row="4" column="2" colspan="2">
           <widget class="QSpinBox" name="p_spinbox_NumberWorkers">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="p_lbl_Severity">
            <property name="text">
             <string>Severity</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="p_lbl_Position">
            <property name="text">
             <string>Position</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0" colspan="3">
           <widget class="QLabel" name="p_lbl_Neglect">
            <property name="text">
             <string>Allow neglect of requirement</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="p_lbl_ID">
            <property name="text">
             <string>ID</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="3">
           <widget class="QDateTimeEdit" name="p_dateTimeEdit_EndDate">
            <property name="wrapping">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="3">
           <widget class="QDateTimeEdit" name="p_dateTimeEdit_StartDate">
            <property name="wrapping">
             <bool>true</bool>
            </property>
            <property name="displayFormat">
             <string>dd.MM.yyyy HH:mm</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="3">
           <widget class="QComboBox" name="p_comboBox_Position"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="p_layout_navigationButtons">
       <item>
        <widget class="QPushButton" name="p_button_CreateNew">
         <property name="text">
          <string>Create new...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="p_button_Open">
         <property name="text">
          <string>Open...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="p_button_Save">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="p_button_SaveAs">
         <property name="text">
          <string>Save as...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>p_line_ID</tabstop>
  <tabstop>p_dateTimeEdit_StartDate</tabstop>
  <tabstop>p_dateTimeEdit_EndDate</tabstop>
  <tabstop>p_comboBox_Position</tabstop>
  <tabstop>p_spinbox_NumberWorkers</tabstop>
  <tabstop>p_spinbox_Severity</tabstop>
  <tabstop>p_checkbox_BiggerAmountWorkers</tabstop>
  <tabstop>p_checkbox_Neglect</tabstop>
  <tabstop>p_list_Requirements</tabstop>
  <tabstop>p_button_AddRequirement</tabstop>
  <tabstop>p_button_DeleteRequirement</tabstop>
  <tabstop>p_button_CreateNew</tabstop>
  <tabstop>p_button_Open</tabstop>
  <tabstop>p_button_Save</tabstop>
  <tabstop>p_button_SaveAs</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
